packages/protocol/contracts/stability/StableToken.sol

```diff
@@ -270,12 +279,12 @@ contract StableToken is
    * @notice Burns StableToken from the balance of msg.sender.
    * @param value The amount of StableToken to burn.
    */
-  function burn(uint256 value)
-    external
-    onlyRegisteredContract(getExchangeRegistryId())
-    updateInflationFactor
-    returns (bool)
-  {
+  function burn(uint256 value) external updateInflationFactor returns (bool) {
+    require(
+      msg.sender == registry.getAddressForOrDie(getExchangeRegistryId()) ||
+        msg.sender == registry.getAddressFor(GRANDA_MENTO_REGISTRY_ID),
```

Comment:
I'm wondering if StableToken should have a governable limit for both Validators and Granda Mento as to how much they can mint? Maybe it's moot but it might limit the impact of a bug in both contracts?

Reply:
I like the idea but am not sure if it fits within the scope of this PR, wdyt?